packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/wdio-services.4/VisualRegressionService.tool.mdx

```diff
@@ -83,10 +83,10 @@ All of these provide options that will help you to capture screenshots in differ
 available:
```

Comment:
Should we remove this VisualRegressionService mdx because everything described here is for internal use?Consumers shouldn't use this directly and this would create more confusion for them.

Reply:
@StephenEsser is working on a doc revamp. I believe he address this question in his PR.